easybuild/easyconfigs/p/PyTorch/PyTorch-1.0.1-fosscuda-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,240 @@
+    'ffe077d0fccc34aa95c2aba7a3abadbdc2a480c44a238879381733d38322e921',  # pybind11-20180911.tar.gz
+    '828e0d6238e2129a9e08071750dc16ba10e38eacf96f21b8a71e501c2085b282',  # clang-cindex-python3-20170330.tar.gz
+    'b84c49b4bdc12b0e8f1bff365dcefd50e5a2a25eeef8c928aeb2dab9fda4d599',  # cpuinfo-20190117.tar.gz
+    'b9520d6ce9f8b9928bf517696ea62fd03308fd21dafdebfec7aeb8894b3813ab',  # onnx-20181204.tar.gz
+    'd293a86f4c82a31990b89be2b9787df045e3f6b86378e9950e5e1145d4a08e6a',  # protobuf-20171113.tar.gz
+    '90ab4f60ae0e99c0172dd5201dccc6de6c8f384a21b3d959588f4c1e00beec0f',  # pthreadpool-20181008.tar.gz
+    '07625551202a1f35fe5e826669ad495962b26f6a139290e90b321c5996f002ef',  # FXdiv-20181016.tar.gz
+    'afd27417c6df1c85f79a68361055dd3adc292174913c39c1f58b698b9cfd7926',  # psimd-20180906.tar.gz
+    'dacd927f345c40163532e65ed3ea796d190f4efdd125838401a35e8147e55313',  # FP16-20181011.tar.gz
+    '02d278559609982f2dbd973f265caf674274ac7a61ddd867c4956176252d940e',  # NNPACK-20180903.tar.gz
+    'f822489cbbf3cd09db578ea7ac9af40a9d3184f8ff95cb98e8b33b71ed5d89f8',  # sleef-20180509.tar.gz
+    '13100c3deed300bbf16f87d8af3539f432462bfef9d38f0c7e3e387dc2e88676',  # PeachPy-20180219.tar.gz
+    'dc0c9ca760bb1bf858dd1aebaf1471b8437c1b2ef9a9b6acaf302fa14b04194f',  # QNNPACK-20181205.tar.gz
+    'd2dabcd172db8fb91dfa75422a5eb6f55f55f22d047d9f1b02eb6a3be88a09cd',  # fbgemm-20190123.tar.gz
+    '0de43b6eaddd356f1d6cd164f73f37faf2f6c96fd684e1f7ea543ce49c1d144e',  # benchmark-20180606.tar.gz
+    'fd9e9f0d6ef407a162f5f283c52d3c271efc1092377274abe1e30dff83dbc1a4',  # eigen-20180908.tar.gz
+    'e12e096e986221d96d55185e870397b3717b625d6c5d155fbd9a7be314bed06a',  # gemmlowp-20180828.tar.gz
+    '72b4835362d8586e315bdf5c30eea99f2efcf14f19162f00ac0675e8b1b22e61',  # nccl-20181109.tar.gz
+    '7224b03af4acbc54525105bb42e3ecd75c66a3a5b47e8a725ab008467c4109f9',  # cub-20170829.tar.gz
+    'c7418025bd57b3143e651433d0874a39629fce6cc96b43e3f8d5861a8ad23ffb',  # ideep-20181224.tar.gz
+    'f98d2c297353acfab1ecff2054bee65efd2367b4c621e762cf26f50e037a9728',  # mkl-dnn-20190131.tar.gz
+    'f00dc3b142a5be399bdeebd7e7ea369545a35d4fb84c86f98b6b048d72685295',  # mklml_lnx_2019.0.1.20180928.tgz
+    'c8dabee9fd710976c6073a81d2cad2735f95c063819cf8effc001e1940f63283',  # PyTorch-1.0.1-fix-findAVX.patch
+    '569333e194d6a1d716972e17ad2ca1cd72e92a8845cee2f02ab2d5f541f013e9',  # PyTorch-1.0.1-fix-for-intel.patch
+]
+
+builddependencies = [('CMake', '3.12.1')]
+
+dependencies = [
+    ('Python', '3.6.6'),
+    ('PyYAML', '3.13', versionsuffix),
+    ('hypothesis', '4.5.0', versionsuffix),
+    ('cuDNN', '7.1.4.18'),
+    ('Ninja', '1.8.2'),
+    ('magma', '2.5.0'),
+]
+
+download_dep_fail = True
+
+# This environment is needed to prevent setup.py from invoking git to determine version.
+prebuildopts = 'PYTORCH_BUILD_VERSION=%(version)s PYTORCH_BUILD_NUMBER=1 VERBOSE=1 '
+prebuildopts += ' LDFLAGS="$LDFLAGS -ldl"'
+preinstallopts = 'VERBOSE=1 LDFLAGS="$LDFLAGS -ldl"'
```

Comment:
Need to set CUDNN_ROOT_DIR too or it won't find the right cuDNN version.

Reply:
done

Reply:
Meh.... that didn't help. I wonder what else we need to do.

Reply:
Looks like setup.py need CUDNN_LIB_DIR, CUDNN_INCLUDE_DIR set in the env.

Should have looked in the right place from the start....

Reply:
Ok, I've made that change.

Reply:
Btw, the commit message should be descriptive of the actual change made.

Reply:
Apparently we need the CUDNN stuff in installopts too.

Reply:
Ok added.

Reply:
Thanks for solving this.